sys/net/include/etx_beaconing.h

```diff
@@ -7,6 +7,7 @@
  */
 
 /**
+ * @defgroup etx_beaconing
  * @ingroup     rpl
  * @{
  * @file        etx_beaconing.h
```

Comment:
Remove the filename from after `@file` (but keep the `@file` command), Doxygen will figure out the filename by itself.